.teamcity/_self/projects/WebApp.kt

```diff
@@ -614,6 +614,27 @@ object Translate : BuildType({
 			"""
 			dockerImage = "%docker_image_e2e%"
 		}
+		bashNodeScript {
+			name = "Notify GlotPress Translate build is ready"
+			scriptContent = """
+				if [[ "%teamcity.build.branch.is_default%" == "true" ]]; then
+					exit 0
+				fi
+
+				curl -X POST https://translate.wordpress.com/api/localci/-relay-new-strings-to-gh \
```

Comment:
I'm not sure if this is needed, but if we need to add some kind of auth key, there is an existing example here: https://github.com/Automattic/wp-calypso/blob/04d0f317a147797deeda9a5223858b4374c542a8/.teamcity/_self/lib/wpcom/WPComPluginBuild.kt#L193-L200

Reply:
The endpoint does not currently require authentication, so I think it's fine to deploy it as is for now.